Makefile

```diff
@@ -27,6 +27,14 @@ ci-lint-image: ## Builds linting container.
 ci-lint: ## Runs linting in linting container.
 	docker run --rm -ti -v /var/run/docker.sock:/var/run/docker.sock securedrop-lint:${TAG}
 
+.PHONY: ci-typelint-image
+ci-typelint-image: ## Builds type linting container.
+	docker build $(EXTRA_BUILD_ARGS) -t securedrop-typelint:${TAG} -f devops/docker/Dockerfile.typeannotation .
+
+.PHONY: ci-typelint
+ci-typelint: ## Runs type linting in container.
+	docker run --rm -ti -v /var/run/docker.sock:/var/run/docker.sock -v ${PWD}:/src securedrop-typelint:${TAG}
```

Comment:
that will not work because the docker container does not run on the same machine as the CI